### ISSUE TITLE
Highlight is off by 1 in Live Preview

### ISSUE THREAD

**Issue by SufianBabri** [lijyze/obsidian-advanced-codeblock#10]
First of all, thank you for this awesome plugin. It is easy to understand and integrate.

## Plugin Specifics
Name: Advanced Codeblock
Version: 1.1.2

## Problem
There's a difference between Live Preview vs Reading Mode if I do not add `nums`.

If I add `jsx {3-5}` to the following code block, the highlight changes in the Live Preview vs Reading View.
```jsx
function Form() {
    const [firstName, setFirstName] = useState("John");
    const [lastName, setLastName] = useState("Smith");
    // ✅ Good: calculated during rendering
    const fullName = firstName + " " + lastName;
    // ...
}
```
Live Preview's screenshot:
![live-preview](https://github.com/lijyze/obsidian-advanced-codeblock/assets/9989266/c0004a74-7e97-44bd-85de-00b51199ccc7)

Reading View's screenshot:
![reading-view](https://github.com/lijyze/obsidian-advanced-codeblock/assets/9989266/7273194f-1fea-4612-b7f9-5554326d3ced)

## Summary
The Live Preview incorrectly starts and ends highlighting 1 line early if we do not pass `nums` in the first line.

**Comment by lijyze** [comment]
I just release a new version.
If it can't fix you problem, feel free to contact me.

**Comment by SufianBabri** [comment]
@lijyze I have upgraded the plugin to version 1.1.4 and the issue is still happening even after relaunching the Obsidian.

**Comment by lijyze** [comment]
Which theme are you using?


在 2023年7月28日，17:32，Sufian ***@***.***> 写道：

﻿

@lijyze<https://github.com/lijyze> I have upgraded the plugin to version 1.1.4 and the issue persists even after relaunching the Obsidian.

**Comment by SufianBabri** [comment]
@lijyze happens with both the default theme and Obsidian Nord.

**Comment by lijyze** [comment]
Bug has been fixed, please try v1.1.5, it should work well.

**Comment by SufianBabri** [comment]
Awesome! Thanks for fixing it.
